aospy/__init__.py

```diff
@@ -1,6 +1,6 @@
 """aospy: management, and analysis of gridded climate data."""
-from .__config__ import (user_path, LAT_STR, LON_STR, PFULL_STR, PHALF_STR,
-                         PLEVEL_STR, TIME_STR, TIME_STR_IDEALIZED, YEAR_STR)
+from .__config__ import user_path
+from . internal_names import *
```

Comment:
Let's omit this.  I think it's fine to access the names through internal_names, and wildcard imports are generally frowned upon.